composer.json

```diff
@@ -9,17 +9,27 @@
         "issues": "https://github.com/odolbeau/phone-number-bundle/issues"
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "phpstan/extension-installer": true
+        }
     },
     "require": {
-        "php": ">=7.4",
+        "php": ">=8.0",
         "giggsey/libphonenumber-for-php": "^8.0",
         "symfony/framework-bundle": "^4.4|^5.3|^6.0",
```

Comment:
We can also remove any support above 5.4

Reply:
As well as the related code 👍